rainier-core/src/main/scala/com/stripe/rainier/core/Model.scala

```diff
@@ -0,0 +1,35 @@
+package com.stripe.rainier.core
+
+import com.stripe.rainier.compute._
+
+case class Model(targets: List[Target]) {
```

Comment:
Is it intentional that RV uses `Set[Target]` while this uses `List[Target]`?

Separately, do we ever expect the user to access `targets` directly? I'm wondering if it should be `private[core]`.

Reply:
Great question, and I believe I should change this to `Set`. The case we're trying to catch here is something like model1.merge(model2).merge(model1) - that's much easier to accidentally do using the old RV API but still possible here and worth preventing.

Agreed on the access control.